package.json

```diff
@@ -30,31 +29,12 @@
   },
   "type": "commonjs",
   "main": "dist/index.js",
-  "module": "dist/esm/index.mjs",
-  "types": "dist/index.d.ts",
```

Comment:
**issue (blocking):** this will not work and result in that typescript cant find the correct types